launch/state_machines/challenge_eegpsr.launch

```diff
@@ -33,6 +33,9 @@
             <param name="robot_name" value="$(arg robot_name)" />
         </node>
 
+        <node pkg="topic_tools" type="relay" args="message_to_ros hmi/string" name="chat_to_hmi"/>
```

Comment:
No idea .. this I add this?

Reply:
I did. This relays the hmi/string to message_to_ros